docs/yaml_docs/pipeline_visualization_yml.md

```diff
@@ -0,0 +1,161 @@
+
+You can download the different ingestion `pipeline.yml` files here:
+- Basic `pipeline.yml` file (not prefilled) that is generated when calling `panpipes vis config`: [Download here](https://github.com/DendrouLab/panpipes/blob/main/panpipes/panpipes/pipeline_vis/pipeline.yml)
+- `pipeline.yml` file for [Visualizing data Tutorial](https://panpipes-tutorials.readthedocs.io/en/latest/visualization/pipeline_yml.html): [Download here](https://panpipes-tutorials.readthedocs.io/en/latest/_downloads/29daa86241829b362152785caf30ab61/pipeline.yml)
+
+## Compute resources options
+<span class="parameter">resources</span><br>
+Computing resources to use, specifically the number of threads used for parallel jobs.
+Specified by the following three parameters:
+  - <span class="parameter">threads_high</span> `Integer`, Default: 1<br>
+        Number of threads used for high intensity computing tasks. 
+        For each thread, there must be enough memory to load all your input files at once and create the MuData object.
+
+  - <span class="parameter">threads_medium</span> `Integer`, Default: 1<br>
+        Number of threads used for medium intensity computing tasks.
+        For each thread, there must be enough memory to load your mudata and do computationally light tasks.
+
+  - <span class="parameter">threads_low</span> `Integer`, Default: 1<br>
+  	    Number of threads used for low intensity computing tasks.
+        For each thread, there must be enough memory to load text files and do plotting, requires much less memory than the other two.
+
+<span class="parameter">condaenv</span> `String` (Path)<br>
+    Path to conda environment that should be used to run panpipes.
+    Leave blank if running native or your cluster automatically inherits the login node environment
+
+## Loading and merging data options
+### Data format
+
+<span class="parameter">sample_prefix</span> `String`, Mandatory parameter, Default: test<br>
+Prefix for the sample that comes out of the filtering/ preprocessing steps of the workflow.
+
+<span class="parameter">mudata_obj</span> `String`, Mandatory parameter <br>
+ Path to the output file from preprocessing (e.g. `../vis/test.h5mu`).
+ Ensure that the submission file is in the right format and that the correct path is provided.
+
+<span class="parameter">modalities</span><br>
+<span class="parameter">rna</span> `Boolean`, Default: True <br>
+<span class="parameter">prot</span> `Boolean`, Default: True <br>
+<span class="parameter">atac</span> `Boolean`, Default: False <br>
+<span class="parameter">rep</span> `Boolean`, Default: True <br>
+<span class="parameter">multimodal</span> `Boolean`, Default: True <br>
+Set the modalities to True or False depending on what is present in the mudata_obj
+
+<span class="parameter">grouping_vars</span> `String`, Default: sample_id  rna:leiden_res0.6 <br>
+On dot plots and bar plots, grouping vars are used to group other features (for categorical, continuous, and feature plots).
+
```

Comment:
can you add a screen shot of how the grouping vars should be added as a list like you do for the plot markers sections below?